docusaurus/docs/code-base-works/sortable-table.md

```diff
@@ -18,3 +18,13 @@ When the user scrolls the page and other columns become visible, Sortable Table
 
 Typically a delayed component would initally not render itself at all - or render a simple indicator that the content has not loaded. When its
 `startDelayedLoading` method is called, it will then render its full content, fetching data as needed etc,
+
+
+### Table header
+
+There are two ways table headers are populated:
+
+- Customize header: You can create a customized header for the sortable table by defining it in `explore.js`. Read more [how to create custom table headers](/code-base-works/customising-how-k8s-resources-are-presented#customising-tables) for each type.
```

Comment:
I would change for something like:

Custom header: You can create a customized header for the sortable table by defining it in  the appropriate configuration file  under `shell/config/product` folder. Read more [how to create custom table headers](/code-base-works/customising-how-k8s-resources-are-presented#customising-tables) for each type.

Reply:
Fixed